BraveShared/tr.lproj/BraveShared.strings

```diff
@@ -0,0 +1,1353 @@
+"AuthenticationImmediately" = "hemen";
+
+/* Error message displayed when user enters incorrect passcode when trying to enter a protected section of the app with attempts remaining */
+"AuthenticationIncorrectAttemptsRemaining" = "Yanlış parola. Yeniden deneyin (Kalan denemeler: %d).";
+
+/* Touch ID prompt subtitle when accessing logins */
+"AuthenticationLoginsTouchReason" = "Girişlere şimdi erişmek için parmak izinizi kullanın.";
+
+/* Error message displayed when user enters incorrect passcode and has reached the maximum number of attempts. */
+"AuthenticationMaximumAttemptsReached" = "Çok fazla başarısız deneme.\n Lütfen %d dakika içinde yeniden deneyin.";
+
+/* Error message displayed when user enters incorrect passcode and has reached the maximum number of attempts. */
+"AuthenticationMaximumAttemptsReachedNoTime" = "Maksimum deneme sayısına ulaşıldı. Lütfen daha sonra yeniden deneyin.";
+
+/* Error message displayed when user enters incorrect passcode and has reached the maximum number of attempts. */
+"AuthenticationMaximumAttemptsReachedOneMinute" = "Çok fazla başarısız deneme.\n Lütfen 1 dakika içinde yeniden deneyin.";
+
+/* Label for the Passcode item in Settings */
+"AuthenticationPasscode" = "Oturum Açma Parolası";
+
+/* Text displayed above the input field when confirming a passcode */
+"AuthenticationReenterPasscode" = "Parolayı yeniden girin";
+
+/* Text displayed in the 'Interval' section, followed by the current interval setting, e.g. 'Immediately' */
+"AuthenticationRequirePasscode" = "Parola İste";
+
+/* Touch ID prompt subtitle when accessing the require passcode setting */
+"AuthenticationRequirePasscodeTouchReason" = "Gerekli parola aralığınızı yapılandırmaya erişmek için parmak izinizi kullanın.";
+
+/* Title of the dialog used to set a passcode */
+"AuthenticationSetPasscode" = "Parola Belirle";
+
+/* When the user taps on the passcode pane, the touch gesture recognizer will bring up the keyboard back on the screen when hidden on iPad. */
+"AuthenticationTouchForKeyboard" = "Klavye erişilebilirliğini getirmek için dokunun";
+
+/* Label for the Touch ID/Passcode item in Settings */
+"AuthenticationTouchIDPasscodeSetting" = "Dokunmatik Kimlik ve Parola";
+
+/* Label used as a setting item to turn off passcode */
+"AuthenticationTurnOffPasscode" = "Parolayı Kapat";
+
+/* Label used as a setting item to turn on passcode */
+"AuthenticationTurnOnPasscode" = "Parolayı Aç";
+
+/* Error message displayed when user enters incorrect passcode when trying to enter a protected section of the app */
+"AuthenticationWrongPasscodeError" = "Yanlış parola. Yeniden deneyin.";
+
+/* Label for Cancel button */
+"AuthPromptAlertCancelButtonTitle" = "İptal";
+
+/* Authentication prompt message with a realm. First parameter is the hostname. Second is the realm string */
+"AuthPromptAlertFormatRealmMessageText" = "@tarafından bir kullanıcı adı ve parola %1$ isteniyor. Site diyor ki: %2$ @";
```

Comment:
Very confused what is happening here honestly. I believe it should be 

"AuthPromptAlertFormatRealmMessageText" = "%@ tarafından bir kullanıcı adı ve parola isteniyor. Site diyor ki: %@";